api/src/repositories/submission-repository.test.ts

```diff
@@ -313,4 +313,43 @@ describe('SubmissionRepository', () => {
       expect(response).to.eql(mockResponse);
     });
   });
+
+  describe('listSubmissionRecords', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('should succeed with valid data', async () => {
+      const mockResponse = {
+        submission_status: 'Submission Data Ingested',
+        submission_id: 1,
+        source: 'SIMS',
+        uuid: '2267501d-c6a9-43b5-b951-2324faff6397',
+        event_timestamp: '2022-05-24T18:41:42.211Z',
+        delete_timestamp: null,
+        input_key: 'platform/1/moose_aerial_stratifiedrandomblock_composition_recruitment_survey_2.5_withdata.zip',
+        input_file_name: 'moose_aerial_stratifiedrandomblock_composition_recruitment_survey_2.5_withdata.zip',
+        eml_source: null,
+        darwin_core_source: 'test',
+        create_date: '2022-05-24T18:41:42.056Z',
+        create_user: 15,
+        update_date: '2022-05-24T18:41:42.056Z',
+        update_user: 15,
+        revision_count: 1
+      };
+      const mockQueryResponse = ({ rowCount: 1, rows: [mockResponse] } as any) as Promise<QueryResult<any>>;
+
+      const mockDBConnection = getMockDBConnection({
+        sql: async () => {
+          return mockQueryResponse;
+        }
+      });
```

Comment:
(Not actually a change thats needed) This could be simplified to 
```typescript
const mockDBConnection = getMockDBConnection({ sql: async () => mockQueryResponse });
```

Reply:
Maybe something we can do as a separate PR.  I want to introduce the import formatter package once all of the current PR's are merged, so maybe as part of that one.